docs-source/usersguide/application.rst

```diff
@@ -38,18 +38,61 @@ Follow these instructions to install OpenMM.  There also is an online
 troubleshooting guide that describes common problems and how to fix them
 (http://wiki.simtk.org/openmm/FAQApp).
 
+
+Installing with Conda
+*********************
+
+There are two ways to install OpenMM: using the Conda package manager, or with
```

Comment:
Should the paragraph "There are two ways to install..." come *before* the "Installing with Conda" subsection?